apps/vpnmanager/sentry.server.config.ts

```diff
@@ -1,8 +1,13 @@
 import * as Sentry from "@sentry/nextjs";
 
-Sentry.init({
+const options = {
   dsn: process.env.SENTRY_DSN,
-  environment: process.env.SENTRY_ENV,
+  environment: process.env.SENTRY_ENV ?? "local",
   tracesSampleRate: 1,
-  debug: false,
-});
+};
+
+Sentry.init(options);
+
+export function initSentry() {
```

Comment:
```
export function initSentry() {
 try {
    Sentry.init(options);
} catch (error) {
    console.error("Failed to initialize Sentry", error);
}
}
``` 
just so to catch any errors that might occur during sentry setup.

Reply:
@VinneyJ What would the relevance of this be apart from logging the error without notifying us?

I'd rather have it as is. If a critical error happens at initialization, let it terminate since it is necessary for its execution.

Reply:
Yes @koechkevin makes sense. Was just looking at it as an alternative notification for the team when Sentry fails to initialize in production for some reason, ensuring we're promptly informed for example: 
```
    } catch (error) {
        console.error("Failed to initialize Sentry", error);
        notifySlack
```
However, the code looks good to me.